packages/coinstac-ui/app/render/state/ducks/util.js

```diff
@@ -1,7 +1,11 @@
 import app from 'ampersand-app';
 
+export const logUI = (level, msg) => {
+  app.renderLogger[level](msg);
```

Comment:
_Also_ also, the render process’s logger still lives in the main process, and log calls are synchronous over RPC. Wonder if there's a better approach for this.